lib/main.js

```diff
@@ -0,0 +1,152 @@
+'use babel'
+
+/* @flow */
+
+import Path from 'path'
+import { CompositeDisposable } from 'atom'
+
+type Linter$Provider = Object
+
+module.exports = {
+  config: {
+    executablePath: {
+      type: 'string',
+      default: Path.join(__dirname, '..', 'node_modules', 'jshint', 'bin', 'jshint'),
+      description: 'Path of the `jshint` executable.'
```

Comment:
We spawn it as Node, not a system executable, otherwise we would be using the `node_modules/.bin` path :wink:.